pkg/minikube/cluster/cluster.go

```diff
@@ -229,11 +229,38 @@ var assets = []fileToCopy{
 	},
 }
 
-func updateLocalkubeFromURL(config KubernetesConfig, client *ssh.Client) error {
+func (k *KubernetesConfig) getLocalkubeCacheFilepath() string {
+	return filepath.Join(constants.Minipath, "cache", "localkube",
+		filepath.Base("localkube-"+util.GetMD5Hash(k.KubernetesVersion)))
```

Comment:
This was done in case the flag passed was a URL as I was not sure if that would cause errors.  This might not be the best option as it will not be obvious which localkube is which version which people might find useful.  I will look into alternatives, thanks.